gammapy/utils/scripts.py

```diff
@@ -67,8 +68,8 @@ def read_yaml(filename, logger=None, checksum=False):
     return data
 
 
-def write_yaml(dictionary, filename, logger=None, sort_keys=True, checksum=False):
-    """Write YAML file.
+def to_yaml(dictionary, sort_keys=False):
```

Comment:
The docstring should be updated because of the change of the parameters...
Should we add an API change decorator?

Reply:
This is a new function here.